### ISSUE TITLE
Is there a bug preventing to broadcast a transaction that calls a function in a smart contract?

### ISSUE THREAD

**Issue by WonkyGiraffe** [paritytech/parity#8753]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Parity version?**:Parity v1.10.4-20180514 stable
> - **Which operating system?**:  Linux
> - **How installed?**: .deb
> - **Are you fully synchronized?**:  yes
> - **Which network are you connected to?**: ethereum
> - **Did you try to restart the node?**:yes
![1](https://user-images.githubusercontent.com/39807899/40799746-16f2ea46-6517-11e8-94da-62d464076b10.png)
![2](https://user-images.githubusercontent.com/39807899/40799750-1aeea414-6517-11e8-9910-8a3339fc5330.png)
![3](https://user-images.githubusercontent.com/39807899/40799755-1e441234-6517-11e8-8bb2-021f9613992b.png)
![5](https://user-images.githubusercontent.com/39807899/40799764-2154c1f8-6517-11e8-99d1-2627d5c6cf19.png)

So basically i submit a 0 ETH transaction that just calls a function in a smart contract. Even with 800k gas it's stuck at broadcasting the transaction since yesterday.

**Comment by Tbaut** [comment]
1 days for a transaction seems prohibitively long indeed :)
You wrote that you are synced but I couldn't tell from your screenshot. What blocks is your node at?
This could be one reason. Your screenshot shows 22Gwei which is definitely enough according to [ethgasstation](https://ethgasstation.info/). Can you see the transaction in Etherscann? in the TxQueueViewer Dapp?

**Comment by WonkyGiraffe** [comment]
![6](https://user-images.githubusercontent.com/39807899/40830723-d50e6140-658f-11e8-9c80-9fae25133a35.png)

It's quite synced :)

I can't see the transaction in etherscan, will check in TxQueueViewer as well shortly

**Comment by WonkyGiraffe** [comment]
Turns out the gas price wasnt enough to enter the queue even if it was in the middle of the parity estimations. 

TxQueueViewer showed the reason, oddly parity has no way to show that. 

Thank you!
